src/pybind/mgr/dashboard/README.rst

```diff
@@ -55,6 +55,12 @@ See the `Ceph Dashboard plugin documentation
 and configure the dashboard manually and how to configure other settings, e.g.
 access to the Ceph object gateway.
 
+Supported browsers
+------------------
+
+We support the current versions of `Chrome <https://www.google.com/chrome/>`_
```

Comment:
Do we have any developers which are using Windows IE and can confirm that the UI properly works? If so, maybe we should add this browser here, too.

Reply:
To my knowledge, @Exotelis uses Windows - he will be back next week to comment.